src/options.c

```diff
@@ -115,6 +115,8 @@ static struct opt_entry opt_list[] = {
 		0, 0, OPT_FMT_STR_ALLOC, &options.charset},
 	{"mask", FLG_MASK_SET, FLG_MASK_CHK,
 		0, 0, OPT_FMT_STR_ALLOC, &options.mask},
+	{"hc-mask-file", FLG_HC_MASK_SET, FLG_HC_MASK_CHK,
+		0, 0, OPT_FMT_STR_ALLOC, &options.hc_mask_file},
```

Comment:
I think it should be --mask-file, not --hc-mask-file.

Same goes for variables and other things. FLG_HC_MASK_CHK should be FLG_MASK_FILE_CHK and so on.